connector/http-sink-connector/README.md

```diff
@@ -0,0 +1,157 @@
+# HTTP sink Connector
+
+In this example, you'll setup the followings:  
+* nodejs as an API server 
+* Confluent Platform with Connect and http sink connector.  
+
+You will install and manage the https sink connector and plugin jars through the declarative `Connector` CRD.  
+In this scenario, we deploy the a Confluent cluster without security.  
+
+## Set up Pre-requisites
+
+Set the tutorial directory for this tutorial under the directory you downloaded
+the tutorial files:
+
+```
+export TUTORIAL_HOME=<Tutorial directory>/connector/http-sink-connector
+```
+
+Create a namespace:  
+```
+kubectl create ns confluent
+```
+
+Deploy Confluent for Kubernetes (CFK): 
+
+```
+helm upgrade --install confluent-operator \
+  confluentinc/confluent-for-kubernetes \
+  --namespace confluent
+```
+
+## Deploy nodejs API server   
+
+```
+kubectl --namespace confluent apply -f $TUTORIAL_HOME/nodejs.yaml
+```
+
+Check that all components were created: 
+
+```
+kubectl --namespace confluent get pods -l app=my-backend-api
+
+NAME                             READY   STATUS    RESTARTS   AGE
+my-backend-api-589cb6bdf-jn96w   1/1     Running   0          48s
+```
+
+## Deploy Confluent Cluster Components
+
+```
+kubectl --namespace confluent apply -f $TUTORIAL_HOME/confluent-platform.yaml
+```
+Check that zookeeper, kafka and connect cluster are deployed:
+
+```   
+kubectl --namespace confluent get confluent
+```
+
+Notice that the connect pod might take a bit longer to create since it's downloading the connector jars from Confluent Hub.  
+
+## Create secret, topic, and producer
+
+Once the Kafka cluster is ready create the secret, topic, and producer, all within one YAML file:    
+```
+kubectl --namespace confluent apply -f $TUTORIAL_HOME/producer-app-data.yaml
+```  
+
+Check topic 
+```
+kubectl --namespace confluent get topic
+```
```

Comment:
could also consume from the topic here

Reply:
The consumer step is part of the validation, I hope that's OK ?